### ISSUE TITLE
Top level project is missing Maven pom.xml file

### ISSUE THREAD

**Issue by simonedavico** [benchflow/faban#3]
Top level project is missing Maven pom.xml file. As a result, when selecting `Import from existing Maven project...` in Eclipse, `common`, `driver` and `harness` are imported as separate projects into the workspace (with errors, see attached images).

<img width="646" alt="screen shot 2015-09-25 at 17 04 11" src="https://cloud.githubusercontent.com/assets/4446853/10103994/93043eb0-63a7-11e5-808f-35a1dd9efc6c.png">
<img width="1440" alt="screen shot 2015-09-25 at 17 04 44" src="https://cloud.githubusercontent.com/assets/4446853/10103993/9303ce6c-63a7-11e5-80cc-7ead2d6e5a58.png">

**Comment by VincenzoFerme** [comment]
The top level project is not a Maven project. We should make it a Maven project and integrate the ant builds utilised by Faban, in the Maven lifecycle.

**Comment by VincenzoFerme** [comment]
@simonedavico is this solved?

**Comment by simonedavico** [comment]
No, not at all. I can do this after I finish the tasks for the 1.3.2 milestones, but it will take me some time to read up on Maven `pom` files and how to seamlessly integrate them with the already existing Ant `build.xml`.
